### PR TITLE
Ps 5.7 myr 35

### PR DESCRIPTION
**Pull request by george-lorch**

### CODE REVIEW

**Comment by george-lorch** [comment]
http://jenkins.percona.com/view/5.7/job/mysql-5.7-param/593/

**Review by laurynas-biveinis — APPROVED** [review]

**Comment by laurynas-biveinis** [comment]
@georgelorchpercona, only 5.6 has been merged before, I merged this one, please make sure bugs etc are updated

**Comment by george-lorch** [comment]
Hm, I specifically did not merge this as the 5.7 release branch for 5.7.17 hasn't been made but the 5.6.35 has, was just trying to keep commits to releases somewhat in sync. It doesn't really matter much I guess.

**Pull request merged**
